siliconcompiler/tools/openroad/sc_floorplan.tcl

```diff
@@ -50,11 +50,11 @@ if {[expr ! [dict exists $sc_cfg "read" def $sc_step $sc_index]]} {
 
     # source tracks from file if found, else else use schema entries
     if [dict exists $sc_cfg pdk aprtech openroad $sc_stackup $sc_libtype tracks] {
-	source [lindex [dict get $sc_cfg pdk aprtech openroad $sc_stackup $sc_libtype tracks] 0]
-    } else {
+	source [lindex [dict get $sc_cfg pdk aprtech openroad $sc_stackup $sc_libtype tracks]]
+    } elseif {$sc_process == "asap7"} {
```

Comment:
Wouldn't be better to provide the make_tracks file to source instead of having a special carveout for asap7?

Reply:
Yeah, tired fix. With the new support for make_tracks, I don't think there is any need to support the grid-->track translation. I removed it.